org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/internal/IdeAttachedProjectConnection.java

```diff
@@ -63,7 +63,6 @@ public TestLauncher newTestLauncher() {
     private <T extends LongRunningOperation> T configureOperation(T operation) {
         BuildEnvironment buildEnvironment = this.delegate.getModel(BuildEnvironment.class);
         this.gradleArguments.applyTo(operation, buildEnvironment);
-        this.gradleArguments.describe(this.progressAttributes, buildEnvironment);
```

Comment:
Unrelated change?

Reply:
It's unrelated but valid: I didn't notice before that at some point we started printing the connection properties into the background console. We only want this behavior for the task execution. This change fixes that problem.